src/svelte-asset.js

```diff
@@ -1,8 +1,10 @@
 const path = require('path');
-const {compile, preprocess} = require('svelte');
 const {Asset} = require('./parcel-adapter');
 const {sanitize, capitalize} = require('./utils');
 
+const {version: svelteVersion} = require('svelte/package.json');
```

Comment:
Perhaps we should extract a major version from this similar to how it works in the commit you linked

Reply:
Good point - I had intended to grab the major version :)